### DISCUSSION TITLE
AMD hints ROCM support for Windows

### DISCUSSION THREAD

**Discussion by eneskuluk** [vladmandic/sdnext#163] [General]
Looks like AMD is going to bring ROCM to windows, even low-mid end consumer cards will be supported: https://www.techpowerup.com/307259/amd-brings-rocm-to-consumer-gpus-on-windows-os
Would it be easy to implement this to the repo?
Though it seems it is still very far away from release.

**Comment by vladmandic** [comment]
its a medium size effort - initial compatibility should be relatively easy, but then its checking where each tensor needs to be relocated.
the problem is i don't have amd system and i cannot do that blindly. i can guide a dev if someone volunteers on what needs to be done, but i cannot do it myself.
